### DISCUSSION TITLE
ImportError: No module named solcx(Linux user)

### DISCUSSION THREAD

**Discussion by 0xsomnus** [smartcontractkit/full-blockchain-solidity-course-py#493] [Q&A]
GM fellow devs. Whenever I try to test and deploy my code, I get the following error despite installing solcx with pip multiple times. (I'm on Linux specifically ubuntu)

`Traceback (most recent call last):
  File "deploy.py", line 1, in <module>
    from solcx import compile_standard, install_solc
ImportError: No module named solcx`

If anyone has had the same issue and fixed it, I'd be grateful for the answer as it's not limited to just this project haha.

**Comment by cromewar** [comment]
GM, have you selected correct the python interpreter on vscode? You should select the one you have installed solcx. Additionally you can use a Python virtual env and install your packages there and of course run the project as well.

**Reply by 0xsomnus** [reply]
I did the first step prior to this and have still had the same error. I'll look into virtual envs for python and seee if that fixes it. Thanks

**Reply by cromewar** [reply]
Pleas let us know if this info was helpful for you.

Best regards, Cromewar.
